File: LeetCode-Problems/Easy/Numbers-Smaller-Than-Current.py
# Python
class Solution(object):
    def smallerNumbersThanCurrent(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        returnedList = list()
        count = 0
        for x in range (0, len(nums)):
            for y in range (0, len(nums)):
                if (nums[x] > nums[y]):
                    count = count + 1
            returnedList.append(count)
            count = 0
        return returnedList

File: LeetCode-Problems/Easy/Numbers-Smaller-Than-Current.java
//Java
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int[] returnedArray = new int[nums.length];
        int count = 0;
        for (int i = 0; i < nums.length; i++){
            count = 0;
            for (int j = 0; j < nums.length; j++){
                if (nums[i] > nums[j]){
                    count++;
                }
            }
            returnedArray[i] = count;
        }
        return returnedArray;
    }
}
